caas/kubernetes/provider/k8swatcher.go

```diff
@@ -25,17 +26,17 @@ type kubernetesNotifyWatcher struct {
 	clock    jujuclock.Clock
 	catacomb catacomb.Catacomb
 
-	out       chan struct{}
-	name      string
-	k8watcher watch.Interface
+	informer cache.SharedIndexInformer
+	name     string
+	out      chan struct{}
```

Comment:
nitpic - less churn to move out back to where it was and put informer here
(and below in newKubernetesNotifyWatcher)